openfecwebapp/templates/partials/navigation.html

```diff
@@ -8,17 +8,17 @@
         </a>
       </li>
       <li class="site-nav__item" data-submenu="data">
-        <a href="{{ url_for('search') }}" class="site-nav__link {% if parent == 'data' %}is-parent{% endif %}">
+        <a href="{{ url_for('search') }}" tabindex="0" class="site-nav__link {% if parent == 'data' %}is-parent{% endif %}">
           <span class="site-nav__link__title">Campaign finance data</span>
         </a>
       </li>
       <li class="site-nav__item site-nav__item--secondary" data-submenu="help">
-        <a href="{{ cms_url }}/help-candidates-committees/" class="site-nav__link">
+        <a href="{{ cms_url }}/help-candidates-committees/" tabindex="0" class="site-nav__link">
```

Comment:
AFAIK it doesn't matter. What matters is that there is one so that the browser knows its focusable, and then they're treated in the order they appear in the DOM.